doc-deploy-changelog/action.yml

```
@@ -262,8 +277,8 @@ runs:
           fi
 
           # Configure git username & email
-          git config user.name 'pyansys-ci-bot'
-          git config user.email '92810346+pyansys-ci-bot@users.noreply.github.com'
+          git config user.name ${{ env.BOT_USER }}
+          git config user.email ${{ env.BOT_EMAIL }}
```

Why not just use input rather than env?
```suggestion
          git config user.name ${{ inputs.bot-user }}
          git config user.email ${{ inputs.bot-email }}
```